rubin_sim/scheduler/utils/sky_area.py

```diff
@@ -542,7 +541,9 @@ def combine_maps(self, trim_overlap=True):
             # 'self.dustfree' is already 0/1 dust-acceptable or not
             with warnings.catch_warnings():
                 warnings.simplefilter('ignore', category=UserWarning)
+                # XXX--magic number
```

Comment:
--- sorry - - see next comment. 
~This doesn't seem consistent with the numbers or code I see on main. 
https://github.com/lsst/rubin_sim/blob/main/rubin_sim/scheduler/utils/sky_area.py#L151 and kwargs in set_dustfree_wfd
Were the changes in the dust-smoothing intentional or has this been picked up from an old branch? 
The numbers in the version on main came from testing various smoothing beam sizes and cutoffs, and testing the effect on the resulting dust-free WFD region (how much in the region was dust-free and how much was 'extra') and is reflected in the SkyCoverage.ipynb notebook. (perhaps could be better documented how/why those numbers were chosen, but the plot after cell 10 here https://github.com/lsst-pst/survey_strategy/blob/main/fbs_2.0/SkyCoverage.ipynb reflected those choices)~
![image](https://user-images.githubusercontent.com/1855711/131571913-f4f50243-d182-471b-bd78-e19aa18b4928.png)

Reply:
Ah, shoot - sorry.  I see I was looking at the wrong part of the code. This is in the 'combine' part. And yes, this was also a bit of trial-and-error, but not really documented similarly.

Reply:
Ok, so maybe I can make some suggestions to replace the 'magic numbers' comments so that there is a bit more traceability.

Reply:
```#XXX - magic numbers (regarding the 3 deg smoothing)``` could be replaced by

```# Use 3 degree smoothing beam here, so that edges are still not too sharp, but are smoothed on the order of the FOV```